### PR TITLE
Receive executeScript result

### PR DESCRIPTION
**Pull request by Doflatango**

### CODE REVIEW

**Comment by Doflatango** [comment]
Related issues #161 #139 #69

File: windows/webview_bridge.cc [hunk 1]
```diff
@@ -428,9 +429,10 @@ void WebviewBridge::HandleMethodCall(
       std::shared_ptr<flutter::MethodResult<flutter::EncodableValue>>
           shared_result = std::move(result);
 
-      webview_->ExecuteScript(*script, [shared_result](bool success) {
+      webview_->ExecuteScript(*script, [shared_result](bool success, LPCWSTR resultObjectAsJson) {
```
Comment (jnschulze) [inline windows/webview_bridge.cc:432]: Please convert the `LPCWSTR` to `std::string` in `webview.cc`.

File: example/lib/main.dart [hunk 2]
```diff
@@ -51,6 +51,19 @@ class _ExampleBrowser extends State<ExampleBrowser> {
       await _controller.setPopupWindowPolicy(WebviewPopupWindowPolicy.deny);
       await _controller.loadUrl('https://flutter.dev');
 
+      print(await _controller.executeScript('document.activeElement.getBoundingClientRect()')); // ❌ return null
```
Comment (jnschulze) [inline example/lib/main.dart:54]: Please migrate all prints to proper tests.

**Review by jnschulze — CHANGES_REQUESTED** [review]

File: example/lib/main.dart (hunk 2 above)
Comment (Doflatango) [reply example/lib/main.dart]: Sorry buy i'm have no time to move those examples to test cases, so i removed them.

**Review by jnschulze — APPROVED** [review]

**Pull request merged**